examples/tour/Cargo.toml

```diff
@@ -9,4 +9,13 @@ publish = false
 iced.workspace = true
 iced.features = ["image", "debug"]
 
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
 tracing-subscriber = "0.3"
+
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+iced.workspace = true
+iced.features = ["image", "debug", "webgl"]
+
+console_error_panic_hook = "0.1"
+console_log = "1.0"
+log.workspace = true
```

Comment:
The `log` dependency here is most likely unnecessary.

Reply:
It is (almost) unnecessary here if I am not mistaken.
`log` is needed to specify the `log::Level` when initializing the `console_log` logger.
```rust
console_log::init_with_level(log::Level::Debug)
```
Correct me if Im missing something tho.

Reply:
Can we use just `init`?

Reply:
Oh yeah sure `init` uses `log::Level::Info` I think that should be fine too.

I wasn't sure what log level would be sensible here in that example.

Should I also change
```rust
console_log::init_with_level(log::Level::Debug)
```
to 
```rust
console_log::init()
```

And remove the `log` dependency in the integration example too?

Reply:
Go for it!

Reply:
Done